cms/djangoapps/contentstore/views/videos.py

```diff
@@ -803,10 +825,10 @@ def videos_post(course, request):
 
         resp_files.append({'file_name': file_name, 'upload_url': upload_url, 'edx_video_id': edx_video_id})
 
-    return JsonResponse({'files': resp_files}, status=200)
+    return {'files': resp_files}, 200
```

Comment:
any specific reason for removing JSON response here?

Reply:
I am allowing the callers of this function (both the original api and my new one) to choose the type of response that they return.